cogs/deepfry.py

```diff
@@ -0,0 +1,65 @@
+from io import BytesIO
+import discord
+import requests
+from PIL import Image, ImageOps, ImageEnhance
+from discord.ext import commands
+from cogs.base import BaseCog
+
+FILE_TYPES = [
+    'bmp',
+    'gif',
+    'jpg',
+    'jpeg',
+    'png'
+]
+
+
+class Colours:
+    RED = (254, 0, 2)
+    YELLOW = (255, 255, 15)
+
+
+class Deepfry(BaseCog):
+    @commands.command()
+    async def deepfry(self, ctx):
+        for attachment in ctx.message.attachments:
+            if check_file_type(attachment.filename):
+                img = fry_to_shits(attachment.url)
+                img.save('deepfry.jpg')
```

Comment:
Where does this save to?

Reply:
On line 51 inside the fry_to_shits command, instead of using BytesIO, use a TemporaryFile so that way garbage collection will have a fallback

Reply:
Done